lib/csa/ccm/cli/resource.rb

```diff
@@ -15,14 +14,6 @@ def self.lookup_version(version)
           Dir["#{root_gem}/resources/**/*v#{version}*.xlsx"]
```

Comment:
I think you will want to use `File.join` and `Pathname` for Windows compatibility.

Reply:
I think the code above should work for windows too, I just added appveyor and it's green

Ruby correctly handle `/` path separators on windows